target/linux/realtek/files-5.10/drivers/clk/realtek/clk-rtl83xx.c

```diff
@@ -74,7 +72,7 @@ static const int rtcl_regs[RTCL_SOCCNT][REG_COUNT][CLK_COUNT] = {
 	}
 
 struct rtcl_reg_set {
-	unsigned int rate;
+	unsigned long rate;
```

Comment:
Why the change? Isn't (unsigned) int the same size as (unsigned) long anyway?

Reply:
wherever rate is used we take the long type. So be consistent here.

Reply:
OK, it appears you changing for consistency with the clock framework, which uses "unsigned long" for clock rates. Please try to put these "why"s in the commit message, so we don't have to spend time wondering why you're making changes like this.